src/components/StyleGuide/Breakpoints.mdx

```diff
@@ -58,4 +59,4 @@ $nypl-max-width: $nypl-breakpoint-xl;
 
 For additional spacing information, please refer to the Figma Main file.
 
-- https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17983%3A60146
+- https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?type=design&node-id=36835-26738&mode=design&t=w3RXt8BBaOjtXvvh-11
```

Comment:
This points to a file that displays the older values of 1280, 960, 600, and 320.

Reply:
Right.  The Figma file still needs to be updated.  I just made some basic adjustments to reclect the new breakpoints, but I will also do a round of comprehensive updates for the VDL.